guide/daily-spending-wallet/backup-and-recovery/cloud-backup.md

```diff
@@ -25,47 +25,45 @@ Illustration sources
+   width = 1600
+   height = 600
+   layout = "full-width"
 %}
 
 # Automatic cloud backup
 
-### How it works
+## How it works
 
-The user's wallet recovery data and channel states are backed up on their respective cloud storage provider. The [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) only needs to be backed up once. However, channel states may need to be backed up frequently because they change every time the user's wallet sends, receives, or routes a payment; therefore, the channel state should be backed up automatically.
+The user’s wallet recovery data and channel states are backed up on their respective cloud storage provider. The [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) only needs to be backed up once. However, channel states need to be backed up frequently because they change every time this wallet sends or receives a payment; therefore, the channel state is backed up in the cloud automatically every time it is changed.
 
-If the user's device is lost, stolen, or broken, they can use the backup from their cloud storage account (such as Apple iCloud or Google Drive) to regain access to their funds. To learn more about the technical details, hop over to the [private key management]({{ '/guide/private-key-management/introduction' | relative_url }}) section on this topic.
+If the user’s device is lost, stolen, or broken, they can use the backup from their cloud storage account to regain access to their funds. This could include common ones like Apple iCloud and Google Drive, open source ones like NextCloud, or even a [Lightning Wallet Server]({{'/guide/how-it-works/lightning-services/#backups' | relative_url}}).  To learn more about the technical details, hop over to the [private key management]({{ '/guide/private-key-management/introduction' | relative_url }}) section on this topic.
 
-> Crypto rookies perceive themselves as “incapable of self-managing keys”.
->
-> <cite>As researched by <a href="https://informationsecurity.uibk.ac.at/pdfs/ECIS21_Non-Adoption_of_Crypto-Assets.pdf">Abramova et al</a></cite>
+This method provides users with the feeling that they can rely on their cloud provider to regain access to their funds rather than putting the responsibility entirely on themselves. It also significantly reduces friction during onboarding.
 
-This method provides users with the feeling that they can rely on their cloud provider to regain access to their funds rather than putting the responsibility on themselves.
-
-Below is an example of what the user interface might look like for an automatic cloud backup that significantly reduces friction during onboarding. An optimal phase to show this may be after the wallet has received funds for the first time, so that the user has an incentive to back up their wallet.
+Below is what this wallet’s automatic cloud backup user interface looks like.
 
 <div class="image-slide-gallery">
 
 {% include picture.html
```

Comment:
This gallery is probably not the best place but we need content on what it would look like in the UI when a backup is made every time a payment is sent or received and what it would look like if that backup failed (maybe cloud provider is down?). Some kind of notification indicating the backup failed and some actions to fix it. 

In the settings -> backup & recovery it could also state "Last backup was made xx/xx/xxxx" for extra assurance.

Reply:
I don't think there should be an overt indication in the UI when a backup successfully happens. It's part of the normal expected functioning of the app -- the user doesn't need to be notified every time the app behaves as expected.

There could be some way to check this in the settings screen. Didn't we discuss doing a Settings section for this app? Maybe that would be a good section to show this?

Displaying an error state could be interesting, though.

Reply:
Added an error handling section.

Reply:
Agree, can have a stub about latest back up in that settings page of which Christoph has opened an issue for